### PR TITLE
myheadexplodez' card

### PR DESCRIPTION
**Pull request by myheadexplodez**
added a new card

### CODE REVIEW

**Comment by CandidDeer** [comment]
I've gone in and added your card manually, since your card's content itself is fine. Have you maybe manually formatted the index.html after adding your card? This may cause content to shift and thereby cause the large diff error (= changes to all 2000+ lines). You may want to avoid formatting manually, at least the final merged code. Ofc it's fine to format only your own card. Anyway, let me know if you need help understanding anything about this concept. Your card will be live in a moment. Thx.

**Comment by myheadexplodez** [comment]
I have seen the lines being slid together , but had no idea for what reason , when i did not touch anything else , only the things we are supposed to touch , meaning only the template to copy, and then paste it to the well shown place.
Now, when i see that i had nearly 2k lines of changes , is something i can not believe , unless the lines have been pushed somewhat 18 lines below where they originally were...
Also , when i press cmd opt J to control, i get an error message , related to JS.
i have saved pictures tho, if you are interested in the error messages, i will attach it/them.
So in conclusion i have not formatted anything , i have even deactivated my Prettier extension before opening the index.html.
I have deleted my first branch , because i wanted to start from scratch , but then the same thing has happened as you could see. I would love to know how and what i have mistaken , when i only followed instructions which were quite clear.
Thank you so much :-)
